### PR TITLE
Remove dual dispatchers docs and CHANGELOG

### LINKED ISSUE #1128: Remove DualDispatchers
**🧐 Motivation**
They are not supported on live networks and make our library more blotted. 

**📝 Details**
We've had `DualDispatchers` for a while now, and since they are not supported on live networks they haven't been adopted. They are an additional liability in our codebase and make the amount of LOC bigger for any security review. 

```[tasklist]
### Tasks
- [x] Remove access dual dispatchers
- [x] Remove account dual dispatchers
- [x] Remove erc20 dual dispatchers
- [x] Remove erc721 dual dispatchers
- [x] Remove erc1155 dual dispatchers
- [ ] docs and CHANGELOG
```

### PR DESCRIPTION
**Pull request by ericnordelo**
Partially fixes #1128

### CODE REVIEW

**Review by andrew-fleming — COMMENTED** [review]
Looking good, Eric! I left some comments and questions

File: docs/modules/ROOT/pages/interfaces.adoc [hunk 1]
```diff
@@ -72,34 +76,25 @@ pub trait ERC20ABI<TState> {
...
 
 === Dispatcher traits
-This is a utility trait to interface with contracts whose interface is unknown. Read more in the xref:#dualcase_dispatchers[DualCase Dispatchers] section.
 
-```cairo
-#[derive(Copy, Drop)]
-pub struct DualCaseERC20 {
-    contract_address: ContractAddress
-}
+Traits decorated with `#[starknet::interface]` automatically generate a dispatcher that can be used to interact with contracts that implement the given interface. They can be imported by appending the `Dispatcher` and `DispatcherTrait` suffixes to the trait name, like this:
```
[the remark this answers is not in the document]
Comment (ericnordelo) [reply docs/modules/ROOT/pages/interfaces.adoc:80]: Added a mention and a link.

File: packages/utils/src/selectors.cairo
Comment (andrew-fleming) [inline packages/utils/src/selectors.cairo:1]: Don't forget to remove the `selectors` entry in utils/README

File: docs/modules/ROOT/pages/interfaces.adoc [hunk 2]
```diff
@@ -72,34 +76,30 @@ pub trait ERC20ABI<TState> {
...
+Other types of dispatchers are also auto-generated from the annotated trait. See the
+{interacting-with-another-contract} section of the Cairo book for more information.
+
+NOTE: In the example, the `IERC20Dispatcher` is the one used to interact with contracts, but the
+`IERC20DispatcherTrait` needs to be in scope for the functions to be available.
+
 == Dual interfaces
 
+WARNING: `camelCase` functions are deprecated and maintained only for Backwards Compatibility. We recommend to
+integrate you contracts and components with the `snake_case` interfaces. The `camelCase` functions will be removed in
```
Comment (andrew-fleming) [inline docs/modules/ROOT/pages/interfaces.adoc:97]:
```suggestion
WARNING: `camelCase` functions are deprecated and maintained only for Backwards Compatibility.
It's recommended to only use `snake_case` interfaces with contracts and components. The `camelCase` functions will be removed in
```

**Review by andrew-fleming — APPROVED** [review]
I left a couple final suggestions. Otherwise, I think we're good to go!

**Pull request merged**